src/components/tooltip/tooltip.test.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import user from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import Tooltip from './tooltip.jsx';
+
+describe('tooltip', () => {
+  const requiredProps = {
+    children: <div id="my-component" aria-describedby="my-component-tooltip">Child</div>,
+    id: 'my-component-tooltip',
+    text: 'Help Text',
+  };
+
+  it('has defaults', () => {
+    render(<Tooltip {...requiredProps} />);
+    expect(screen.getByText('Child').parentElement).toMatchSnapshot();
+
+    user.hover(screen.getByText('Child'));
+    expect(screen.getByText('Help Text')).toBeInTheDocument();
```

Comment:
We don't need this if we're snapshotting?

Reply:
0e2461fc